src/Configuration/Dotenv/Types.hs

```diff
@@ -30,4 +36,29 @@ defaultConfig =
     { configExamplePath = []
     , configOverride = False
     , configPath = [ ".env" ]
+    , configVerbose = False
     }
+
+
+newtype ReaderT r m a = ReaderT { runReaderT :: r -> m a }
+mapReaderT :: (m a -> n b) -> ReaderT r m a -> ReaderT r n b
+mapReaderT f m = ReaderT $ f . runReaderT m
```

Comment:
@FranzGB Are we using this function? If not, can we remove it?

Reply:
We are using this function on the instance `Functor fmap` of the ReaderT

Reply:
I remove the export should I remove then instance as well?

Reply:
Let's not remove the instance. Thanks!